SPECS/chrony/chrony.spec

```diff
@@ -79,11 +79,15 @@ EOF
 # - use Windows NTP server (time.windows.com) instead of a pool
 # - enable leapsectz to get TAI-UTC offset and leap seconds from tzdata
 # - enable keyfile
+# - enable virtual PHC
 sed -e 's|^pool.*|server time.windows.com|' \
     -e 's|#\(leapsectz\)|\1|' \
     -e 's|#\(keyfile\)|\1|' \
         < examples/chrony.conf.example2 > chrony.conf
 
+echo -e "\n# Enable virtual PTP clock source." >> chrony.conf
+echo -e "refclock PHC /dev/ptp_hyperv poll 3 dpoll -2 offset 0" >> chrony.conf
```

Comment:
I'm concerned about the implications of including this by default in every chrony.conf.
I agree with setting this in Azure images but it'll impact our edge baremetal and VM configs if it's here.

Reply:
Will fix.

Reply:
Done, though I can't reasonably test this.